tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -994,6 +1014,59 @@ public function userGetsAllTheMembersOfGroupUsingTheGraphApi($user, $group): voi
 		$this->featureContext->setResponse($this->listGroupMembers($group, $user));
 	}
 
+	/**
+	 * @When user :user retrieves all groups along with their members using Graph API
+	 * @When user :user gets all the members information of group :group using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $group
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userRetrievesMembersOfAllGroupsUsingTheGraphAPI(string $user, string $group = ''): void {
+		$this->featureContext->setResponse($this->listAllGroupsAlongWithAllMemberInformation($user, $group));
+	}
+
+	/**
+	 * @Then the group :group should have the following member information
+	 *
+	 * @param string $group
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function theGroupShouldHaveTheFollowingMemberInformation(string $group, TableNode $table): void {
```

Comment:
we could use the new system of checking JSON responses to check the group information. What do you think?

Reply:
we have a separate issue and currently not implemented on ocis (on todo) https://github.com/owncloud/ocis/issues/3179. so lets not include that to this PR.